doc/3rd_party_licenses/cpptoml/LICENSE

```diff
@@ -0,0 +1,18 @@
+Copyright (c) 2014 Chase Geigle
```

Comment:
Does the Eclipse [handbook](https://www.eclipse.org/projects/handbook/) require to copy the license files of the dependecies? Would a link be sufficient?

Reply:
There is no direct statement about that in the handbook. The intention here is to be compliant and make sure that the third-party dependencies and their licenses are good visible. One thing is to mention them in the NOTICE file and additionally to deploy the corresponding licenses. Other OSS projects like Mozilla or Chromium do it the same way and go even further that they deploy all 3rd_party repos completely in their repos.